deploifai/dataset/info.py

```diff
@@ -3,57 +3,78 @@
 from deploifai.context import (
     pass_deploifai_context_obj,
     DeploifaiContextObj,
-    project_found,
+    is_authenticated,
+    dataset_found
 )
 
+from deploifai.utilities.frontend_routing import get_dataset_route
+
 
 @click.command()
 @pass_deploifai_context_obj
-@project_found
+@is_authenticated
+@dataset_found
 def info(context: DeploifaiContextObj):
     """
-    Get information about a dataset
+    Get information about the initialized dataset
     """
     deploifai_api = context.api
 
-    storage_details = deploifai_api.get_data_storage_info(data_storage_id)
+    dataset_id = context.dataset_config["DATASET"]["id"]
+
+    storage_details = deploifai_api.get_data_storage_info(dataset_id)
 
-    project_id = storage_details["project"]["id"]
+    project_name = storage_details["projects"][0]["name"]
+    workspace_name = storage_details["account"]["username"]
     cloud_provider = storage_details["cloudProviderYodaConfig"]["provider"]
     container = storage_details["containers"][0]
-    container_name = container["directoryName"]
+    container_name = container["cloudName"]
 
-    if cloud_provider == "AWS":
-        click.secho("AWS dataset CLI support is coming soon.", fg="blue")
+    click.secho("Workspace Name: {}".format(workspace_name), fg="blue")
+    click.secho("Project Name: {}".format(project_name), fg="blue")
+    click.secho("Deploifai Storage ID: {}".format(dataset_id), fg="blue")
+    click.secho("Dataset Status: {}".format(storage_details["status"]))
 
-    elif cloud_provider == "AZURE":
-        storage_account_name = storage_details["cloudProviderYodaConfig"][
-            "azureConfig"
-        ]["storageAccount"]
+    url = get_dataset_route(workspace_name, project_name, dataset_id)
+    click.secho(
+        "\nAccess the dataset on deploifai through: \n {}\n".format(url),
+        underline=True,
+        fg="green",
+    )
 
-        click.secho("Deploifai Storage ID: {}".format(data_storage_id), fg="green")
+    if cloud_provider == "AWS":
+        click.secho("AWS Storage Account", bold=True, fg="green", underline=True)
+        click.secho("Container Name: {}".format(container_name), fg="green")
         click.secho(
-            "https://deploif.ai/dashboard/{username}/projects/{project_id}/datasets/{data_storage_id}\n".format(
-                username=context.global_config["AUTH"]["username"],
-                project_id=project_id,
-                data_storage_id=data_storage_id,
-            ),
+            "You could use the AWS CLI to list all the blobs in your container on AWS:",
```

Comment:
I think it's unnecessary to give aws, azure, gcp cli instructions. The user should be using `deploifai data [command]`s to interact with their dataset. The `container['cloudName']` should only be a marker for the user to find their container when they access from their cloud service portals.